validator/tests/test_validation.py

```diff
@@ -518,49 +518,47 @@ def test_validation_ccia_ref(self):
         self.check_results(new_run)
         self.delete_run(new_run)
 
-    #     @pytest.mark.long_running
-    #     def test_validation_smap_ref(self):
-    #         run = generate_default_validation()
-    #         run.user = self.testuser
-
-    #         run.reference_configuration.dataset = Dataset.objects.get(short_name=globals.SMAP)
-    #         run.reference_configuration.version = DatasetVersion.objects.get(short_name=globals.SMAP_V5_PM)
-    #         run.reference_configuration.variable = DataVariable.objects.get(short_name=globals.SMAP_soil_moisture)
-    #         run.reference_configuration.filters.add(DataFilter.objects.get(name='FIL_ALL_VALID_RANGE'))
-
-    #         run.reference_configuration.save()
-
-    #         run.interval_from = datetime(2017, 1, 1, tzinfo=UTC)
-    #         run.interval_to = datetime(2018, 1, 1, tzinfo=UTC)
-    #         # different window is used here, because for the default one there is too much memory needed to create and save
-    #         # plots
-    #         run.min_lat = 20.32
-    #         run.min_lon = -157.47
-    #         run.max_lat = 21.33
-    #         run.max_lon = -155.86
-
-    #         run.save()
-
-    #         for config in run.dataset_configurations.all():
-    #             if config != run.reference_configuration:
-    #                 #                 config.filters.add(DataFilter.objects.get(name='FIL_C3S_FLAG_0'))
-    #                 config.filters.add(DataFilter.objects.get(name='FIL_ALL_VALID_RANGE'))
-    #             config.save()
-
-    #         run_id = run.id
-
-    #         ## run the validation
-    #         val.run_validation(run_id)
-
-    #         new_run = ValidationRun.objects.get(pk=run_id)
-    #         print(new_run)
-    #         assert new_run
-
-    #         assert new_run.total_points == 15, "Number of gpis is off"
-    #         assert new_run.error_points == 0, "Too many error gpis"
-    #         assert new_run.ok_points == 15, "OK points are off"
-    #         self.check_results(new_run)
-    #         self.delete_run(new_run)
+    # @pytest.mark.long_running
```

Comment:
Forgot to uncomment this

Reply:
You mean the long running decorator?

Reply:
yes ;) but its fine, I will merge this PR last (after the cdf)

Reply:
done